components/xsd-fu/python/ome/modeltools/language.py

```diff
@@ -347,6 +361,7 @@ def __init__(self, namespace, templatepath):
         self.primitive_type_map[namespace + 'double'] = 'double'
         self.primitive_type_map[namespace + 'anyURI'] = 'std::string'
         self.primitive_type_map[namespace + 'hexBinary'] = 'std::string'
+        self.primitive_type_map['base64Binary'] = 'std::string'
```

Comment:
Depending upon the outcome of the comment above regarding having Base64 in the API, this might need adjusting.  Ideally it would use the VariantPixelBuffer type, but that would be a layering violation; I'd like to move this out of the ome::files library into a separate repository to allow reuse in the lower levels.  Maybe it would be better as a model primitive, with PixelBuffer in ome::common or another separate nd library?